LAD-AMA-Common/telegraf_utils/telegraf_name_map.py

```diff
@@ -36,6 +36,10 @@
 "% Idle Time" : {"plugin":"cpu", "field":"usage_idle", "module":"processor"},
 "% Privileged Time" : {"plugin":"cpu", "field":"usage_system", "module":"processor"},
 
+# VM Insights
+"Processor\UtilizationPercentage" : {"plugin":"cpu_vmi", "field":"Processor\UtilizationPercentage", "module":"processor_vmi"},
```

Comment:
Why is the field naming scheme different for these two entries?

Reply:
cpu, mem and disk don't need rate so they are directly routed to mdsd/me. diskio and net use basic stats so they have slightly different naming because basic stats code seems to hard code and look for certain plugin/counter names